Resources/views/Collector/redis.html.twig

```diff
@@ -102,6 +102,25 @@
             <p{% if profiler_markup_version == 1 %} style="font-style:italic;"{% endif %}>No commands were executed or the logger is disabled.</p>
         </div>
     {% else %}
+        <div class="metrics">
+            <div class="metric">
+                <span class="value">{{ collector.commandcount }}</span>
+                <span class="label">Queries</span>
+            </div>
+
+            <div class="metric">
+                <span class="value">{{ '%0.2f'|format(collector.time) }} <span class="unit">ms</span></span>
+                <span class="label">Query time</span>
+            </div>
+
+            {% if collector.erroredCommandsCount > 0 %}
```

Comment:
I think we should still show **0 Failed Queries**

Reply:
Good suggestion 👍

I've updated the view